src/js/helpers/cleanData.js

```diff
@@ -55,14 +55,16 @@ export const cleanRelatedServiceLinks = (links) => {
   });
 };
 
-export const cleanServiceLinks = (links) => {
+export const cleanLinks = (links, pathPrefix) => {
```

Comment:
haha, I thought you didn't like pathPrefix.

Reply:
I don't like it showing up in multiple places since it's more prone to errors or being missed if we update the path. If we build the links in one place, it's cool.